docs/manual/applications.qbk

```diff
@@ -187,4 +187,50 @@ available, such as for instance to provide your own entry point function instead
 of [funcref hpx_main `hpx_main`]. Please refer to the function documentation for
 more details (see: [headerref hpx/hpx_start.hpp `hpx/hpx_start.hpp`]).
 
+[heading:flexible Suspending and resuming the __hpx__ runtime]
+
+In some applications it is required to combine __hpx__ with other runtimes. To
+support this use case __hpx__ provides two functions: [funcref hpx::suspend
+`hpx::suspend`] and [funcref hpx::resume `hpx::resume`]. [funcref hpx::resume
```

Comment:
This looks like a typo. Did you mean to write `[funcref hpx::suspend` at the end of this line?

Reply:
Line 193? Yes, that's intentional. Maybe the linebreak looks odd in the middle of the funcref thing but boostbook handles that well.

Reply:
No, I'm pretty sure you meant suspend and not resume at the end of this line. Suspend blocks, resume doesn't.

Reply:
Essentially, what's written is this:

`[funcref hpx::resume `hpx::suspend`] is a blocking call which will wait for all scheduled tasks to finish executing and then put the thread pool OS threads to sleep.`